x-pack/plugins/osquery/public/scheduled_query_groups/form/validations.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import { ValidationFunc } from '../../shared_imports';
+
+const idPattern = /^[a-zA-Z0-9-]+$/;
```

Comment:
is this the proper pattern we would like to use for ids? are there any other special characters we should allow?

Reply:
we should allow for `_` as well